### PR TITLE
Bevy Input Docs : gamepad.rs

### PR DESCRIPTION
**Pull request by tguichaoua**
# Objective

Complete the documentation of `bevy_input` (#3492).

This PR is part of a triptych of PRs :
- https://github.com/bevyengine/bevy/pull/9467
- https://github.com/bevyengine/bevy/pull/9468

## Solution

Add missing documentation on items in `gamepad.rs`

### CODE REVIEW

File: crates/bevy_input/src/gamepad.rs [hunk 1]
```diff
@@ -100,6 +106,7 @@ impl Gamepad {
     reflect(Serialize, Deserialize)
 )]
 pub struct GamepadInfo {
+    /// The name of the gamepad.
```
Comment (killercup) [inline crates/bevy_input/src/gamepad.rs:109]: What is an example of a name? Just so people know what to expect.

File: crates/bevy_input/src/gamepad.rs [hunk 2]
```diff
@@ -1054,22 +1065,27 @@ pub struct GamepadConnectionEvent {
...
 impl GamepadConnectionEvent {
+    /// Creates a [`GamepadConnectionEvent`].
     pub fn new(gamepad: Gamepad, connection: GamepadConnection) -> Self {
         Self {
             gamepad,
             connection,
         }
     }
 
+    /// Is the gamepad connected.
```
Comment (killercup) [inline crates/bevy_input/src/gamepad.rs:1076]:
```suggestion
    /// Is the gamepad connected?
```

File: crates/bevy_input/src/gamepad.rs [hunk 3]
```diff
@@ -1054,22 +1065,27 @@ pub struct GamepadConnectionEvent {
...
     pub fn connected(&self) -> bool {
         matches!(self.connection, GamepadConnection::Connected(_))
     }
 
+    /// Is the gamepad disconnected.
```
Comment (killercup) [inline crates/bevy_input/src/gamepad.rs:1081]:
```suggestion
    /// Is the gamepad disconnected?
```

**Comment by killercup** [comment]
Great work on adding docs :) I really like seeing this!

File: crates/bevy_input/src/gamepad.rs [hunk 4]
```diff
@@ -1242,54 +1269,54 @@ const ALL_AXIS_TYPES: [GamepadAxisType; 6] = [
...
     pub const STRONG_MAX: Self = GamepadRumbleIntensity {
         strong_motor: 1.0,
         weak_motor: 0.0,
     };
 
-    /// Creates a new rumble intensity with weak motor intensity set to the given value
+    /// Creates a new rumble intensity with weak motor intensity set to the given value.
     ///
-    /// Clamped within the 0 to 1 range
+    /// Clamped within the 0 to 1 range.
```
[the remark this answers is not in the document]
Comment (tguichaoua) [reply crates/bevy_input/src/gamepad.rs:1309]: Good catch 😉

File: crates/bevy_input/src/gamepad.rs [hunk 5]
```diff
@@ -1054,22 +1065,27 @@ pub struct GamepadConnectionEvent {
...
     }
 
+    /// Is the gamepad disconnected.
     pub fn disconnected(&self) -> bool {
         !self.connected()
     }
 }
 
+/// Gamepad event for when the "value" on the axis changes
+/// by an amount larger than the threshold defined in [`GamepadSettings`].
```
[the remark this answers is not in the document]
Comment (tguichaoua) [reply crates/bevy_input/src/gamepad.rs:1092]: I don't see how to make the docs more clear 🤔 
Any suggestion ?

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**